### PR TITLE
Params v20

### PR DESCRIPTION
**Pull request by magik6k**
- [x] Update FFI

### CODE REVIEW

File: build/paramfetch.go [hunk 1]
```diff
@@ -83,8 +80,8 @@ func (ft *fetch) maybeFetchAsync(name string, info paramFile) {
 			return
 		}
 
-		ft.fetchLk.Lock()
-		defer ft.fetchLk.Unlock()
+		/*ft.fetchLk.Lock()
```
Comment (whyrusleeping) [inline build/paramfetch.go]: why?

Comment (magik6k) [reply build/paramfetch.go]: Wanted to see if it would make fetching faster

File: cli/params.go [hunk 2]
```diff
@@ -10,17 +10,13 @@ var fetchParamCmd = &cli.Command{
 	Name:  "fetch-params",
 	Usage: "Fetch proving parameters",
 	Flags: []cli.Flag{
-		&cli.BoolFlag{
```
Comment (dignifiedquire) [inline cli/params.go]: why did you remove this option?

Comment (magik6k) [reply cli/params.go]: If you don't set `proving-params` you only get verify keys now
